AnimateSvg.js

```diff
@@ -15,20 +14,71 @@ export default function AnimateSvg(props) {
   });
   const sizeAnim = useRef(new Animated.Value(1)).current;
 
-  const { toggle } = useIsShake();
+  const { isShakeTriggered, setIsToggleReady } = props.useIsShake;
+  const { hasToggledBefore } = props.toggledBefore;
 
   useEffect(() => {
     Animated.timing(fadeAnim, {
       toValue: 1,
       duration: SVG_LOAD_DURATION,
       useNativeDriver: true,
-    }).start();
+    }).start(({ finished }) => {
+      setIsToggleReady(true);
```

Comment:
After animation has completed isToggleReady from useIsShake hook is set to true so that future shakes may be registered.